lib/constants.js

```diff
@@ -2,11 +2,16 @@
  * Copyright (c) 2018-2021 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
+const {CONTEXT_URL: ZCAP_CONTEXT_URL} = require('zcap-context');
```

Comment:
We always leave whitespace after `'use strict';`:

```suggestion
'use strict';

const {CONTEXT_URL: ZCAP_CONTEXT_URL} = require('zcap-context');
```

Reply:
Fixed 83499c0